templates/email/properties.js

```diff
@@ -311,25 +313,44 @@ module.exports = {
   /*	                   Social Media Follow                       */
   /* ============================================================= */
   // uncomment this section to customize the list of available Social Networks
-  // images must be relatvie to this file. We recommend to place them in the 'static' folder
-  // socialMediaFollow: [
-  //   {
-  //     name: 'Web',
-  //     img: css.url(__dirname, 'static', 'Website.png'),
-  //     url: 'https://www.example.com'
+  // custom images must be relative to this file. We recommend to place them in the 'static' folder
```

Comment:
Maybe add something like `The path to the 'static' folder can be specified with the 'staticImgPath' constant. To do this, the constant 'const staticImgPath' must be uncommented above.`